test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -71,120 +59,212 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 		gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
 	})
 
-	ginkgo.BeforeEach(func() {
```

Comment:
Haven't made any changes to these test logics, just move them under the `ginkgo.Context` for current tests setting up too many instances/variables I don't need, also `ginkgo.Context` can group them together. 
New added tests from L196.